scripts/check_regression.sh

```diff
@@ -36,7 +37,19 @@ compare_stat_p90() {
 
 calculate_threshold() {
     local past_value="$1"
-    awk -v past="$past_value" 'BEGIN { print past * 1.1 }'
+    awk -v past="$past_value" 'BEGIN { print past * 1.5 }'
```

Comment:
Should we reduce this back to 10%?

Reply:
I think we can go to 15 % instead of 50%, yeah I can make this change